pppd/lcp.c

```diff
@@ -67,12 +72,18 @@
 
 static void lcp_delayed_up(void *);
 
+#define LCP_RTT_MAGIC 0x19450425
+#define LCP_RTT_HEADER_LENGTH 4
+#define LCP_RTT_FILE_SIZE 8192
```

Comment:
There are Linux machines which use a 64kB page size, so we need to either make this a multiple of 64kB or else use getpagesize() in lcp_rtt_open_file(). (Hopefully Solaris has getpagesize().)

Reply:
Why this? The Linux mmap(2) man page explicitly describes what happens when the length is not a multiple of the page size, and I could not find anything forbidding this in the Solaris man page either.

Reply:
You're right, it's OK if length is less than the page size, so ignore this comment.